### PR TITLE
Reformat Playlist PR for New OneClick Code Refactor

### PR DESCRIPTION
**Pull request by Parapass**
peepobruh

### CODE REVIEW

File: ModAssistant/Classes/External Interfaces/BSaber.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    Playlist playlist = JsonSerializer.Deserialize<Playlist>(File.ReadAllText(Path.Combine(Utils.BeatSaberPath, PlaylistFolder, filename)));
+                    foreach (Playlist.Song song in playlist.songs)
+                    {
+                        if (string.IsNullOrEmpty(song.key))
+                        {
+                            await BeatSaver.GetFromHash(song.hash);
+                        }
+                        else
+                        {
+                            await BeatSaver.GetFromHash(song.key);
```
Comment (megalon) [inline ModAssistant/Classes/External Interfaces/BSaber.cs:32]: I believe this should be
`await BeatSaver.GetFromKey(song.key)`

Comment (Parapass) [reply ModAssistant/Classes/External Interfaces/BSaber.cs:32]: PepeLaugh sorry am dumb

**Comment by Assistant** [comment]
I already got Playlists implemented since you asked me to take you out of you're misery, just haven't committed yet.

**Comment by Parapass** [comment]
Thank you for taking my misery away. Should I close this PR?

**Pull request closed without merging**